plugins/topology/src/utils/transform-utils.ts

```diff
@@ -0,0 +1,266 @@
+import {
+  EdgeModel,
+  Model,
+  NodeModel,
+  NodeShape,
+} from '@patternfly/react-topology';
+import * as _ from 'lodash';
```

Comment:
```suggestion
import { get} from 'lodash';
```

Reply:
Thanks, am working on refactoring will replace lodash usage with native methods wherever applicable and prefer named import over all as suggeted